students/uss_tomasz/lesson_12_debugging/12_3_valdated_user_base/validate_user_base.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import csv
+import logging
+import os
+import re
+import sys
+
+
+EMAIL_REGEX = re.compile(r'''(
+    [a-zA-Z0-9._%+-]+  
+    @
+    [a-zA-Z0-9.-]+  
+    .
+    \.[a-zA-Z]{2,4}
+    )''', re.VERBOSE)
+PASSWORD_REGEX = re.compile(r'''
+        (?=.*[A-Z]) # positive look ahead assertions
+        (?=.*[0-9])
+        (?=.*[a-z])
+        .{8}
+        ''', re.VERBOSE)
+TELEPHONE_REGEX = re.compile(r'''(
+    (\+\d{2}\s)?
+    (
+    \d{3}
+    [\s-]?
+    \d{3}
+    [\s-]?
+    \d{3}
+    )
+    )''', re.VERBOSE)
+ZIP_CODE_REGEX = re.compile(r'\d\d-\d\d\d')
+REGEX_DATABASE = {'email': EMAIL_REGEX, 'password': PASSWORD_REGEX, 'telephone': TELEPHONE_REGEX,
+                  'zip_code': ZIP_CODE_REGEX}
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser('Collect database of mail, pass, tel and zip inside csv.')
+    parser.add_argument('-e', action='store', dest='email', help='Email of user. This is unique filed for database.')
+    parser.add_argument('-p', action='store', dest='password', help='Password of user.')
+    parser.add_argument('-t', action='store', dest='telephone', default='000-000-000', help='Telephone of user.')
+    parser.add_argument('-z', action='store', dest='zip_code', default='00-000', help='Zip code of user.')
+    received_args = parser.parse_args(args)
+    if received_args.email is None or received_args.password is None:
+        return {}
+    else:
+        results = {'email': received_args.email, 'password': received_args.password,
+                   'telephone': received_args.telephone, 'zip_code': received_args.zip_code}
```

Comment:
why reassigning? if you need a `dict` instead of `Namespace` instance see `vars` use above